Configurations/commands.js

```diff
@@ -957,6 +957,50 @@ const commands = {
 			perm: "none",
 		},
 	},
+
+	cli: {
+		/*
+		__base: {
+			description: "Hi there!",
+			usage: "[optional] <required>",
+			isMultiline: false,
+		}*/
+		shutdown: {
+			description: "Shuts down the bot.",
+		},
+		restart: {
+			description: "Restarts the bot.",
+		},
+		help: {
+			description: "Gives help.",
+		},
+		leval: {
+			description: "Evaluates something in the command's context.",
+			usage: "[code]",
+			isMultiline: true,
+		},
+		seval: {
+			description: "Evaluates something in the shard context.",
+			usage: "[shard] <code>",
+			isMultiline: true,
+		},
+
+		addmaintainer: {

```

Comment:
Hmm, looks even harder than expected; the fact I have to check so it doesn't remove a sudo maintainer that I'm promoting from normal maintainer list.

Reply:
Because ~~I'm lazy enough to find a solution~~ can't really make a solution on my side it would be easier to merge them, name the command `maintainer` and make them choose if the said maintainer is going to be added or removed.